src/algebra/group_power.lean

```diff
@@ -647,3 +649,41 @@ end int
 
 @[simp] lemma neg_square {α} [ring α] (z : α) : (-z)^2 = z^2 :=
 by simp [pow, monoid.pow]
+
+@[simp] lemma pow_of_add [add_monoid A] (x : A) (n : ℕ) :
+  (multiplicative.of_add x)^n = multiplicative.of_add (n • x) := rfl
```

Comment:
Most of the time the simp-lemmas go the other way, that is, they move the `of_add` into the operation.  I don't know which direction is best.

Reply:
Oh, I missed this comment before. Yes, this definitely goes against the grain of the usual simp normal form.

Reply:
You can think that you move `of_add` outside or `pow` inside. I'm not sure which point of view is more appropriate here. In all my use cases I want to rewrite this lemma left to right but it's OK with me to remove it from `simp` completely and use `simp [pow_of_add]`.

Reply:
OTOH, I move `to_add` inside of `*` in another lemma. I'll see if rewriting in the other direction closes the goals.